src/main/scala/cdevel.scala

```diff
@@ -2,7 +2,7 @@ package ohnosequencesBundles.statika
 
 import ohnosequences.statika._, bundles._, instructions._
 
-case object cdevel extends Bundle() {
+abstract class Cdevel extends Bundle() {
```

Comment:
this doesn't need to be abstract, because you cannot set a version or anything else for this bundle. so better to leave it `case object`.